compose-ui/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.jetbrains.compose)
+}
+
+@OptIn(org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi::class)
+kotlin {
+    targetHierarchy.default()
+    jvm()
+
+    androidTarget {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "1.8"
+            }
+        }
+    }
+
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach {
+        it.binaries.framework {
+            baseName = "compose-ui"
+        }
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(project(":shared"))
+
+                // Compose Libraries
+                implementation(compose.ui)
+                implementation(compose.foundation)
+                implementation(compose.material)
+
+                // Decompose Libraries
+                implementation(libs.decompose.decompose)
+                implementation(libs.decompose.extensionsComposeJetbrains)
+            }
+        }
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
```

Comment:
We don't have any tests yet, but when I add one with the annotation `@kotlin.test.Test` it won't resolve the annotation if the dependency is not included. Should I still remove it because there are no tests (yet)?

Reply:
Interesting! Never seen this issue before. Let's remove it for now, and add it on demand later.

Reply:
> Let's remove it for now, and add it on demand later.

Can do that. :)

> Never seen this issue before.

Eventually this is because you are applying the dependency in the `gradle-setup-plugin` [here](https://github.com/arkivanov/gradle-setup-plugin/blob/2571f348ff73c679c3942cad76b469d3a0327ceb/src/main/java/com/arkivanov/gradle/SetupMultiplatform.kt#L27C30-L27C30) for Decompose and never had to add it explicitly. However, there are cases where `kotlin-test` is not available in Decompose test classes (e.g. inside of [`jvmTest/ChildrenTest`](https://github.com/arkivanov/Decompose/blob/cbb7509686cf21b98e4e40e489c5568783c37a95/extensions-compose-jetbrains/src/jvmTest/kotlin/com/arkivanov/decompose/extensions/compose/jetbrains/stack/ChildrenTest.kt),`org.junit.Test` is used instead).